src/main/resources/cdm/v5/tableCheck_V5_3.sql

```diff
@@ -75,7 +73,7 @@ SELECT TOP 1 specimen_id, person_id, specimen_concept_id, specimen_type_concept_
 FROM @cdm_schema.specimen;
 SELECT TOP 1 visit_detail_id, person_id, visit_detail_concept_id, visit_start_date, visit_start_datetime, visit_end_date, visit_end_datetime, visit_type_concept_id, provider_id, care_site_id, admitting_source_concept_id, discharge_to_concept_id, preceding_visit_detail_id, visit_source_value, visit_source_concept_id, admitting_source_value, discharge_to_source_value, visit_detail_parent_id, visit_occurrence_id
 FROM @cdm_schema.visit_detail;
-SELECT TOP 1 visit_occurrence_id, person_id, visit_concept_id, visit_start_date, visit_start_datetime, visit_end_date, visit_end_datetime, visit_type_concept_id, provider_id, care_site_id, visit_source_value, visit_source_concept_id, admitting_source_concept_id, admitting_source_value, discharge_to_concept_id, discharge_to_source_value, preceding_visit_occurrence_id
+SELECT TOP 1 visit_occurrence_id, person_id, visit_concept_id, visit_start_date, visit_start_datetime, visit_end_date, visit_end_datetime, visit_type_concept_id, provider_id, care_site_id, visit_source_value, admitting_source_concept_id, admitting_source_value, discharge_to_concept_id, discharge_to_source_value, preceding_visit_occurrence_id
```

Comment:
I see the `visit_source_concept_id` in the 5.3.0 DDL - https://github.com/OHDSI/CommonDataModel/blob/v5.3.0/PostgreSQL/OMOP%20CDM%20postgresql%20ddl.txt#L311

Reply:
You are right, have restored it.